### PR TITLE
Rm filepaths from automation files 

### LINKED ISSUE #54: Update `automation` slides
Update `automation` slides:

- Rm filepaths from all files, if any
- Rm user id on email addresses, if any

### LINKED ISSUE #56: Replace and rm old gitignore files

### PR DESCRIPTION
**Pull request by AntoniaChroni**
Please review all files in the `Automation-Reproducibility-compbio` course, and point out if there are any filepaths, or user id on email addresses, if any.

Closes #54.

Here I am also addressing the issue with the gitignore files. Closes #56.

### CODE REVIEW

File: courses/Automation-Reproducibility-compbio/resources/08-how-to-run-docker-image-from-HPC-with-HPC-registry.md
Comment (sharonfreshour) [inline courses/Automation-Reproducibility-compbio/resources/08-how-to-run-docker-image-from-HPC-with-HPC-registry.md:1]: At lines 30 and 158 (166 old) should username for the singularity login be removed?

**Review by sharonfreshour — CHANGES_REQUESTED** [review]
01-c-Best-practices-automation-for-scientists.pdf slides show user ID for path for where to clone repo to (slides labeled 60 and 61)
01-c-Best-practices-automation-for-scientists.pdf slide 66, screenshot of files shows user ID for files

05-install-docker-manually.md, screenshot for step (4) create account still has user ID in email address

File: courses/Automation-Reproducibility-compbio/resources/08-how-to-run-docker-image-from-HPC-with-HPC-registry.md
Comment (AntoniaChroni) [reply courses/Automation-Reproducibility-compbio/resources/08-how-to-run-docker-image-from-HPC-with-HPC-registry.md:1]: @sharonfreshour This is fixed now, see ddbf0c374c720289fcef5fc8202689b9a4da5ab5.

**Comment by AntoniaChroni** [comment]
> 01-c-Best-practices-automation-for-scientists.pdf slides show user ID for path for where to clone repo to (slides labeled 60 and 61)
> 01-c-Best-practices-automation-for-scientists.pdf slide 66, screenshot of files shows user ID for files

This has been addressed here 4470fbe83b3bf0207f1112aba8e4f2ecaec535a9.

**Comment by AntoniaChroni** [comment]
> 05-install-docker-manually.md, screenshot for step (4) create account still has user ID in email address

This was replaced on the image but it wasn't properly updated on the md file. It is now fixed here fc2fd6318667f9998a793a3718dc20ef1bffdb51

**Review by sharonfreshour — APPROVED** [review]
Looks good

**Pull request merged**